CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 21.09
+
+* Added a reason code in order to know why a call was ended
```

Comment:
CHANGELOG file serves to inform API users what changes are done
To be relevant, you must specify which API it's affect and what is the new field name. No need to add description of the field, the openapi documentation will care of it

Reply:
Unless i missed something, this attribute only affect the `call_ended `event. I fixed it and added which attributes was added (`reason_code`) and also added it affects the `call_ended` event.

Reply:
Unless i missed something, this attribute only affect the `call_ended `event. I fixed it and added which attributes was added (`reason_code`) and also added it affects the `call_ended` event.

Reply:
Forget my openapi comment (last line), the implementation introduced my in error (I'll add comment on the right code section)
Your changelog need to be more explicit on which event is affected: ex: 
```
* The event `call_ended` now send `reason_code` to know why a call was ended`.
```